_docs/examples/provider/README.md

```diff
@@ -25,33 +32,45 @@ mkdir akash-demo && cd akash-demo
 
 ### Kubernetes Cluster
 
-* A Kubernetes cluster with `kubectl` connected to it.
-* A route-able IP address for ingress on the cluster.
-* A "star" domain pointing to the cluster route-able IP.
+* Version: `1.19+`
+* A Kubernetes cluster with your local `kubectl` with admin authentication and authorization.
+* A wildcard DNS registry pointing to the cluster route-able IP. eg: `*.akashian.io`
```

Comment:
Maybe we should just go ahead and change this to indicate "a wildcard A Record resolving to the cluster route-able IP"

Reply:
:100: ty